doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,353 @@
+# <a name="top"></a>List of HTTP Headers used by Orion
+
```

Comment:
The "source" of this file should be mentioned. Something like this:

> This document describes all the headers used by Orion, as defined in src/lib/rest/HttpHeaders.h file.

Reply:
Fixed in 12b17b7